frontend/server/src/Controllers/Problem.php

```diff
@@ -3729,6 +3730,11 @@ public static function apiList(\OmegaUp\Request $r) {
         $offset = null;
         $rowcount = \OmegaUp\Controllers\Problem::PAGE_SIZE;
 
+        $onlyQualitySeal = null;
+        $onlyQualitySeal = isset(
+            $r['only_quality_seal']
+        ) ? $r['only_quality_seal'] : null;
```

Comment:
Puedes cambiar todo eso por:
```suggestion
        $onlyQualitySeal = $r->ensureOptionalBool('only_quality_seal');
```
? es equivalente, además de que detecta el tipo correcto del parámetro.

Otra cosa: ¿hay diferencia entre mandar `only_quality_seal` en falso o no mandarlo? Si no hay diferencia, podrías cambiar esa línea por: 
```
$onlyQualitySeal = $r->ensureOptionalBool('only_quality_seal') ?? false;
``` 
y dejar de aceptar `null` en el parámetro 15 de `getList`?

Reply:
Ahora que lo pienso no hay diferencia, mejor lo cambio.